integration/spark/shared/src/test/java/io/openlineage/spark/agent/OpenLineageRunEventTest.java

```diff
@@ -41,10 +41,12 @@ void testSerializeRunEvent() throws IOException, URISyntaxException {
 
     UUID runId = UUID.fromString("5f24c93c-2ce9-49dc-82e7-95ab4915242f");
     OpenLineage.RunFacets runFacets =
-        ol.newRunFacets(
-            ol.newParentRunFacet(
-                ol.newParentRunFacetRun(runId), ol.newParentRunFacetJob("namespace", "jobName")),
-            null);
+        ol.newRunFacetsBuilder()
+            .parent(
+                ol.newParentRunFacet(
+                    ol.newParentRunFacetRun(runId),
+                    ol.newParentRunFacetJob("namespace", "jobName")))
+            .build();
```

Comment:
did you want to add use of this facet in this test case?

Reply:
Added.